backend/.env.example

```diff
@@ -1,3 +1,7 @@
+# Required properties
 INFER_GPT_MODEL="azureai"
-INFER_GPT_MISTRAL_LARGE_ENDPOINT=MISTRAL_LARGE_ENDPOINT
+INFER_GPT_MISTRAL_LARGE_URL=MISTRAL_LARGE_ENDPOINT
 INFER_GPT_MISTRAL_LARGE_KEY=MISTRAL_LARGE_API_KEY
+
+# Optional properties (with default values)
+# INFER_GPT_HOST="8650"
```

Comment:
Strictly speaking this the port (not the host). I think we'd want both in the configuration anyway...

Could I also suggest that we don't comment them out to prove they work successfully.

```
# Optional properties (with default values)
INFER_GPT_HOST="http://localhost"
INFER_GPT_PORT="8650"
```

Reply:
This has been renamed as `INFER_GPT_FRONTEND_URL` and fixed